asv/commands/find.py

```diff
@@ -128,10 +135,29 @@ def do_benchmark(i):
 
             env.install_project(conf, repo, commit_hash)
 
+            params = dict(machine_params.__dict__)
+            params['python'] = env.python
+            params.update(env.requirements)
```

Comment:
This is probably correct but seems quite strange to copy a __dict__. What is going on there ?

Reply:
Add this as comment in the code and ship it ;-)